pkg/server/service.go

```diff
@@ -268,3 +276,21 @@ func (c *criContainerdService) getDeviceUUID(path string) (string, error) {
 func imageFSPath(rootDir, snapshotter string) string {
 	return filepath.Join(rootDir, fmt.Sprintf("%s.%s", plugin.SnapshotPlugin, snapshotter))
 }
+
+// startProfilingServer start http server to profiling via web interface
+func startProfilingServer(addr string, port string) {
+	if ip := net.ParseIP(addr); ip != nil {
+		endpoint := net.JoinHostPort(addr, port)
+		mux := http.NewServeMux()
+		mux.HandleFunc("/debug/pprof/", pprof.Index)
+		mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+		mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+		mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+		err := http.ListenAndServe(endpoint, mux)
+		if err != nil {
+			glog.Errorf("Failed to start server: %v", err)
+		}
+	} else {
+		glog.Warningf("Ignore Start Profiling Server for invalid IP Address:%q", addr)
```

Comment:
Why capitalize?

Reply:
I am sorry for that. Will pay attention to this issue.

Reply:
BTW, do you think use "Error" is better than "Warning"?

Reply:
Should be an error. Actually I don't even think we need to enforce this to be an ip. Hostname should also be ok.

Reply:
Done